snapshot/manager_root_test.go

```diff
@@ -1,3 +1,5 @@
+//+build root
```

Comment:
Build tags aren't really the right mechanism. I'd recommend using a test flag to enable and disable tests that require root. Just declare it with the std pkg and it will show up in the `go test` output.